### ISSUE TITLE
'Apptentive.h' file not found apptentive-react-native

### ISSUE THREAD

**Issue by stephenmclaughlin** [apptentive/apptentive-react-native#133]
When using apptentive-react-native for react native looks like RNApptentiveModule.m is looking for Apptentive.h but that file does not exist. Looks like the file is actually named ApptentiveMain.h. I changed the import in RNApptentiveModule.m to reflect this is. After doing so it builds.

```
#import "RNApptentiveModule.h"
#import "Apptentive.h"

static NSString *const kRejectCode = @"ApptentiveModule";
extern ApptentiveLogLevel ApptentiveLogLevelFromString(NSString *level);


@interface RNApptentiveModule ()
```

changed to

```
#import "RNApptentiveModule.h"
#import "ApptentiveMain.h"

static NSString *const kRejectCode = @"ApptentiveModule";
extern ApptentiveLogLevel ApptentiveLogLevelFromString(NSString *level);


@interface RNApptentiveModule ()
```

**Comment by CaseyApptentive** [comment]
Hi @stephenmclaughlin, thanks for the report. We're looking into this. 

A few questions:
- What version of the React Native SDK are you seeing this on?
- Is this the first time your app has used the Apptentive SDK, or are you migrating from an older version? If so, what older version?

It sounds like the change you made may have worked. Is it up and running now?

Thanks again.

**Comment by stephenmclaughlin** [comment]
@CaseyApptentive 
"react-native": "npm:react-native-tvos@0.62.2-0"
"apptentive-react-native": "5.6.0"
apptentive-ios (~> 5)
#define kApptentiveVersionString @"5.3.1"

This is the first time our app has used apptentive sdk. We just integrated with apptentive a few weeks ago. The change is working. Just wanted to report the issue.

Thanks,
Stephen

**Comment by CaseyApptentive** [comment]
Thank you!

**Comment by weeeBox** [comment]
Fixed in 5.6.1
